### ISSUE TITLE
Make website for pylenium.io

### ISSUE THREAD

**Issue by ElSnoMan** [ElSnoMan/pyleniumio#177]
Our docs are currently hosted at https://docs.pylenium.io, but we need a website for https://pylenium.io

**Comment by gleekzorp** [comment]
Are you wanting to do a rework or just host it somewhere else?

**Comment by gleekzorp** [comment]
After diving deeper into [FastAPI](https://fastapi.tiangolo.com/) I noticed they use [MkDocs](https://www.mkdocs.org/) with [Material Theme](https://squidfunk.github.io/mkdocs-material/).  I'm not sure how much different that is compared to GitBook.

**Comment by ElSnoMan** [comment]
They look pretty similar, but we'll stick with GitBook for now since we have it already and it's working well for us. This task was more about a general website for `https://pylenium.io` since we don't have one.

I've already purchased the domain and have our GitBook Documentation at `https://docs.pylenium.io`.

The question here is:

> Should we invest time in creating the Pylenium website or should we just use a no-code solution like SquareSpace or Webflow?

**Comment by vernko** [comment]
Well...haven’t used Webflow. So not sure about that platform. Square space
is okay. It makes it so can get something up quick and easy, editing is
kind of pain. Unless it’s text/basic content, that’s actually really nice
haha. Like when I’ve made updates to qap.dev haha.

There are limitations though, so my feeling is build one haha (been wanting
to get the rust off my feature building dev), but having time to get it
done. So I don’t know which one we think overall.

**Comment by ElSnoMan** [comment]
Yeah, and that's why I'm leaning towards building one as well. So, what if we got the community involved and did a Hackathon? Best site wins @vernko @gleekzorp

**Comment by gleekzorp** [comment]
Sounds like fun, count me in.  My weekly dev meetup is tomorrow, I can see if anyone else would like to join.
